docs/src/getting_started/fit_simulation.md

```diff
@@ -167,101 +215,186 @@ u0 = [1.0, 1.0]
+Observe how `Optimization.solve` brings the model closer to the experimental data as it iterates towards better ODE parameter values!
+
+Note that we are using the `PolyOpt()` solver choice here which is discussed https://docs.sciml.ai/Optimization/dev/optimization_packages/polyopt/ since parameter estimation of non-linear differential equations is generally a non-convex problem so we want to run a stochastic algorithm (Adam) to get close to the minimum and then finish off with a quasi-newton method (L-BFGS) to find the optima.
 Together, this looks like:
 
 ```@example odefit
-adtype = Optimization.AutoForwardDiff()
-optf = Optimization.OptimizationFunction((x, p) -> loss(x), adtype)
+# Set up the optimization problem with our loss function and initial guess
+adtype = AutoForwardDiff()
 pguess = [1.0, 1.2, 2.5, 1.2]
-optprob = Optimization.OptimizationProblem(optf, pguess)
+optf = OptimizationFunction((x, _) -> loss(x), adtype)
+optprob = OptimizationProblem(optf, pguess)
+
+# Optimize the ODE parameters for best fit to our data
+pfinal = solve(optprob,
+               PolyOpt(),
+               callback = callback,
+               maxiters = 200)
+α, β, γ, δ = round.(pfinal, digits=1)
 ```
 
-Now we solve the optimization problem:
+!!! note
 
-```@example odefit
-result_ode = Optimization.solve(optprob, PolyOpt(),
-                                callback = callback,
-                                maxiters = 200)
-```
+    When referencing the documentation for DifferentialEquations.jl and Optimization.jl
+    simultaneously, note that the variables `f`, `u`, and `p` will refer to different quantities.
+
+    DifferentialEquations.jl:
+
+    ```math
+    \frac{du}{dt} = f(u,p,t)
+    ```
+
+    - `f` in `ODEProblem` is the function defining the derivative `du` in the ODE.
+
+        Here: `lotka_volterra!`
+
+    - `u` in `ODEProblem` contains the state variables of `f`.
+
+        Here: `x` and `y`
+
+    - `p` in `ODEProblem` contains the parameter variables of `f`.
+
+        Here: `\alpha`, `\beta`, `\gamma`, and `\delta`
+
+    - `t` is the independent (time) variable.
+
+        Here: indirectly defined with `tspan` in `ODEProblem` and `saveat` in `solve`
+
+    Optimization.jl:
+
+    ```math
+    \min_{u} f(u,p)
+    ```
+
+    - `f` in `OptimizationProblem` is the function to minimize (optimize).
+
+        Here: the [anonymous function](https://docs.julialang.org/en/v1/manual/functions/#man-anonymous-functions) `(x, _) -> loss(x)`
+
+    - `u` in `OptimizationProblem` contains the state variables of `f` to be optimized.
+
+        Here: the ODE parameters `\alpha`, `\beta`, `\gamma`, and `\delta` stored in `p`
+
+    - `p` in `OptimizationProblem` contains any fixed
+    [hyperparameters](https://en.wikipedia.org/wiki/Hyperparameter_(machine_learning)) of `f`.
 
-and the answer from the optimization is our desired parameters.
+        Here: our `loss` function does not require any hyperparameters, so we pass `_` for this `p`.
```

Comment:
This line needs changed now to be consistent; `_` was changed to `p` above.

I used `_` and wrote this section in an attempt to avoid the confusion I had between this `p` and  `p = [\alpha, \beta, \gamma, \delta]` which is actually passed as `u`.

Reply:
And this just underwent a change anyways. @Vaibhavdixit02 can you update this as part of that whole update?

Reply:
Should instead say something like " Here: our `loss` function does not require any hyperparameters, so we do not provide the `p` argument to `OptimizationProblem`."